clusterloader2/cmd/clusterloader.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/perf-tests/clusterloader2/pkg/config"
+	"k8s.io/perf-tests/clusterloader2/pkg/framework"
+	"k8s.io/perf-tests/clusterloader2/pkg/test"
+)
+
+var (
+	clientConfigPath = flag.String("kubeconfig", "", "path to the kubeconfig file")
+	testConfigPath   = flag.String("testconfig", "", "path to the test config file")
+)
+
+func validateFlags() []error {
+	errList := make([]error, 0)
+	if *clientConfigPath == "" {
+		errList = append(errList, fmt.Errorf("no kubeconfig path specified"))
+	}
+	if *testConfigPath == "" {
+		errList = append(errList, fmt.Errorf("no test config path specified"))
+	}
+	return errList
+}
+
+func main() {
+	flag.Parse()
+	if errList := validateFlags(); len(errList) > 0 {
+		fmt.Printf("Parsing flags error: %v\n", errors.NewAggregate(errList).Error())
+		return
+	}
+
+	f, err := framework.NewFramework(*clientConfigPath)
+	if err != nil {
+		fmt.Printf("Framework creation error: %v\n", err)
+		return
+	}
+	c, err := config.ReadConfig(*testConfigPath)
+	if err != nil {
+		fmt.Printf("Config materializing error: %v\n", err)
```

Comment:
"materializing error" sounds not too clear. I'd suggest replacing with "Config reading error".

Reply:
done